snuba/datasets/configuration/replays/storages/replays.yaml

```diff
@@ -190,19 +190,19 @@ allocation_policies:
         - project_id
       default_config_overrides:
         is_enforced: 0
-  - name: BytesScannedWindowAllocationPolicy
```

Comment:
For my understanding, what's the reason for removing this? Would it be conflicting with the `BytesScannedRejectingPolicy`?

Reply:
no it's just not really useful to have both of them. We're going to be deprecating the BytesScannedWindow policy 

https://www.notion.so/sentry/Increasing-visibility-into-the-Capacity-Management-System-c16b375ed2ee43c19cd2fa163fc93332?pvs=4#185824a1f34b4ff6b31fe1c6ea872aca

Reply:
Just thought I'd hit two birds with one stone. Especially since that policy did not prevent load spikes anyways

Reply:
Makes sense, thanks!